submissions/vovan-zt/Memory-Pair-Game/script.js

```diff
@@ -0,0 +1,128 @@
+﻿'use strict'
+
+const cards = [
+    { name: 'asuma', image: 'img/13.jpg' },
+    { name: 'hinata', image: 'img/14.jpg' },
+    { name: 'naruto', image: 'img/15.jpg' },
+    { name: 'gaara', image: 'img/16.jpg' },
+    { name: 'kakashi', image: 'img/17.jpg' },
+    { name: 'yamato', image: 'img/18.jpg' },
+]
+
+cards.push(...cards)
+
+const sortingCards = (dataBase) => {
+    dataBase.sort(() => {
+        return 0.5 - Math.random()
+    })
+}
+
+const container = document.querySelector('.container')
+
+const generateCards = ({ name, image }) => {
+    return `
+            <div class ="hide-swap">
+                <div class="c1" data-card='${name}'></div>
+                <img class="c2" src="${image}" alt="${name}">
+            </div>
+        `
+}
+
+const showCards = (dataBase) => {
+    sortingCards(cards)
+    const cardsDiv = document.createElement('div')
```

Comment:
Don't write data type in variable names, it's redundant and does not bring any transparency. First of all, you've created a `div` at the same line. Second, modern IDE and code editors will show the type.

So just `card` will be enough.